src/mumble/Settings.cpp

```diff
@@ -335,6 +335,7 @@ Settings::Settings() {
 	bUserTop = true;
 
 	bWhisperFriends = false;
+	iMessageLimitUsers = 0;
```

Comment:
```suggestion
	iMessageLimitUsers = 20;
```
This seems like a more sensible default than `0`.